README.md

```diff
@@ -30,6 +30,13 @@ You'll also need to clone a new `.env` file from the `.env.template` to store lo
 $ cp .env.template .env  # (first time only)
 ```
 
+In the .env file you will need to add a configuration with your own trello details:
```

Comment:
Nice and clear, but it would be good to also add these keys to the `.env.template` as people are very good at skipping past these instructions